src/types/widget.ts

```diff
@@ -284,7 +284,17 @@ export type TransformItems<TItem, TMetadata = TransformItemsMetadata> = (
  */
 export type SortBy<TItem> =
   | ((a: TItem, b: TItem) => number)
-  | Array<'count' | 'isRefined' | 'name:asc' | 'name:desc'>;
+  | Array<
+      | 'count'
+      | 'count:asc'
+      | 'count:desc'
+      | 'name'
+      | 'name:asc'
+      | 'name:desc'
+      | 'isRefined'
+      | 'isRefined:asc'
+      | 'isRefined:desc'
```

Comment:
we could avoid a missed item in here like this:

```ts
type Direction<TCriterion extends string> =
  | TCriterion
  | `${TCriterion}:asc`
  | `${TCriterion}:desc`;

type x = Direction<'count' | 'name' | 'isRefined'>;
```

Reply:
Good idea, addressed in 841cb7ec.